pages/index/index.jsx

```diff
@@ -70,13 +68,17 @@ class Index extends Component {
 
   handleDate(e) {
     const newValue = e.target.value;
-    console.log(newValue);
     this.setState({ date: newValue });
   }
 
   render() {
-    const { classes, latestData, timestamp, formattedLineData, allLineData } = this.props;
-    const state = this.state;
+    const { classes,  formattedLineData, allLineData, width } = this.props;
+    const { currentLine, arrivalWindow, date } = this.state;
+    const data = date === 'Yesterday' ?
+      allLineData[allLineData.length - 2] :
```

Comment:
I think we should work on the API design so that we can select objects by date in future. This works for now though!